packages/eslint-config/README.md

```diff
@@ -22,17 +30,32 @@ Create an `.eslintrc` file in the folder you need to lint.
 }
 ```
 
+## How to contribute
+
+* Make sure that you update the version in the package.json according to the semver-level of your update
+* Make sure this document (especially the [Profiles](#profiles) part) is up-to-date with your contribution
+* Breaking any of the configuration or changing an exported configuration's name makes your update a breaking version
+* As there are two configs, please fill the [changelog](#changelog) below so that it's easier for other devs to find out which modifications break their configuration
+
+## Changelog
+
+* __2.0.0:__
+	* CodeceptJS config removed, the tests should be run with `noGlobals` attribute, the globals are no longer required
+	* The file `api-test-e2e` was removed, the file `api-test` is now generic, the file `api-test-jasmine` replaces the previous `api-test` file with Jasmine conf
+* __1.0.0:__ Initial contribution
+
+
 ## Profiles
 
-There are multiple profiles available for now. The idea is to merge them eventually.  
-The following table indicated what value to set to refers to each profile. 
+There are multiple profiles available for now. The idea is to merge them eventually.
+The following table indicated what value to set to refers to each profile.
 
 | Extends 										| Description 									|
 |-----------------------------------------------|-----------------------------------------------|
 | `@talend/eslint-config` 						| Default Talend configuration 					|
-| `@talend/eslint-config/api-app.yaml` 			| Api team, javascript base configuration 		|
-| `@talend/eslint-config/api-app-react.yaml` 	| Api team, react applications configuration 	|
-| `@talend/eslint-config/api-app-angular.yaml` 	| Api team, Angular applications configuration 	|
-| `@talend/eslint-config/api-test.yaml` 		| Api team, test configuration 					|
-| `@talend/eslint-config/api-test-e2e.yaml` 	| Api team, e2e test configuration 				|
-| `@talend/eslint-config/packaging.yaml` 		| Api team, packaging script configuration 		|
+| `@talend/eslint-config/api-app.yaml` 			| API team, javascript base configuration 		|
+| `@talend/eslint-config/api-app-react.yaml` 	| API team, react applications configuration 	|
+| `@talend/eslint-config/api-app-angular.yaml` 	| API team, Angular applications configuration 	|
+| `@talend/eslint-config/api-test.yaml` 		| API team, test configuration 					|
+| `@talend/eslint-config/api-test-jasmine.yaml` 	| API team, unit tests configuration 				|
```

Comment:
Not sure that `API team, test configuration` and `API team, unit tests configuration` are clear.
Perhaps
* API team, generic test configuration
* API team, jasmine specific test configuration

Reply:
Agreed but what do you mean exactly by generic? Is it unit and E2E compliant or just a basic js support for tests not even enough to support jasmine?
- API team, test base configuration ?